src/items/themes/bouncer.json

```diff
@@ -0,0 +1,30 @@
+  "system": {
+    "type": "",
+    "abilityMod": {
+      "ability": "cha",
+      "mod": 1
+    },
+    "abilityMods": {
+      "parts": []
+    },
+    "critical": {
+      "parts": []
+    },
+    "damage": {
+      "parts": []
+    },
+    "description": {
+      "chat": "",
+      "unidentified": "",
+      "value": "<p>From the vibrant nightlife of Absalom Station to the glitzy casinos of the Burning Archipelago, there are places throughout the galaxy where people gather to party and relax. But you know how much work goes into securing these venues. From fights on the dance floor to grifters trying to hustle, your job has been to protect patrons and hosts by any means necessary. You have sharp situational awareness and know how to defuse an escalating situation efficiently and quietly, even if it means leaving a few bruises.</p><h2 class=\"title\">THEME KNOWLEDGE (1ST)</h2><p>You’ve spent years working in clubs, so long that you can feel when someone is in trouble before trouble starts. You know how to keep tabs on them and when it’s time to escort them off the premises, but you also know when you need to simply apply a little pressure to get the job done. Reduce the DC of a Sense Motive check to determine if someone is lying by 5. Intimidation is a class skill for you, though if it is also a class skill from the class you take at 1st level, you instead gain a +1 bonus to your Intimidation checks. In addition, you gain an ability adjustment of +1 to Charisma at character creation.</p><h2 class=\"title\">TAKING OUT THE TRASH (6TH)</h2><p>You have a talent for finding hidden weapons and other illicit items, as well as subduing those who refuse to follow the rules. It takes you half as much time as others to search an area using Perception, and you gain a +2 bonus on Perception checks opposing Sleight of Hand checks to hide objects, palm objects, or pick pockets. You only take a –2 penalty to your attack roll when you use a weapon that deals lethal damage to deal nonlethal damage instead.</p><h2 class=\"title\">PLAN AHEAD (12TH)</h2><p>You know that having a plan for how to handle people is just as important as actually handling them, and work to impart that understanding on your party. Once per day you can spend 10 minutes observing a location. For the next hour, you and your allies gain a +2 bonus to initiative and can still act on their initiative counts in the surprise round even if they failed a Perception check, but only to take the total defense action.</p><h2 class=\"title\">SITUATIONAL AWARENESS (18TH)</h2><p>You use the same skills you learned protecting clubs to protect your allies. Once per day, when your allies are taking a 10 minute rest, you can choose to guard the party. By doing this, you do not gain the benefits of a 10 minute rest, but you do restore 1 Resolve Point. In addition, once per day when an enemy draws a weapon outside of combat, you recover 1 Resolve Point as a reaction.</p>"
```

Comment:
Some HTML tags in these themes have errant classes from copying from AoN (E.g `h2 ckass="title"`) which should be removed. Running cook should remove most of these.